tests/integration/ambient/baseline_test.go

```diff
@@ -714,9 +714,9 @@ metadata:
   name: policy-waypoint
 spec:
   targetRefs:
-  - kind: Gateway
-    group: gateway.networking.k8s.io
-    name: waypoint
+  - kind: Service
```

Comment:
We will probably want to add a test that if we have 2 services for 1 waypoint, we can apply  a policy to a svc and it impacts ONLY that one